app/helpers/patient_query_helper.rb

```diff
@@ -686,14 +686,16 @@ def advanced_filter_relative_date(patients, field, filter, tz_diff, type)
       timespan = filter[:value][:number].to_i.months if filter[:value][:unit] == 'months'
       return patients if timespan.nil?
 
+      filter_option = advanced_filter_options(current_user).find { |option| option[:name] == filter[:name] }
```

Comment:
I got an error on this line when executing a export with the filter, it doesn't appear like `current_user` is defined within this scope.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1239/commits/ce44712b3eae647097cc66bf795e1581a575d3b0